orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteTaskHandler.kt

```diff
@@ -94,9 +94,9 @@ class CompleteTaskHandler(
       return true
     }
 
-    if (originalStatus == FAILED_CONTINUE) {
-      // the task explicitly returned FAILED_CONTINUE and _should_ run subsequent tasks
-      return false
+    if (status == FAILED_CONTINUE) {
+      // the task explicitly returned FAILED_CONTINUE and _should_ not run subsequent tasks
```

Comment:
Can we reword this comment to make it clearer what's going on?

Something like `the task failed but the stage explicitly set FAILED_CONTINUE, so we _should not_ run subsequent tasks`